feature/preview/src/test/java/com/google/jetpackcamera/feature/preview/PreviewViewModelTest.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2023 The Android Open Source Project
+ * Copyright (C) 2025 The Android Open Source Project
```

Comment:
nit: you don't need to update copyright years whenever modifying a file. The copyright year should reflect the year of inception.